docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -1252,12 +1252,12 @@ or being killed.
 
 Query Parameters:
 
--   **dockerfile** - path within the build context to the Dockerfile. This is 
+-   **dockerfile** - path within the build context to the Dockerfile. This is
```

Comment:
Unfortunately, they're part of the Markdown syntax and can be used to force a line break. Some of them are intended, some are just left-overs, so it's always tricky to remove them automatically